tests/testthat/test-sumTable.R

```diff
@@ -42,7 +42,8 @@ test_that("sumTable gives a error when at least one alpha = 0", {
       parY = c(1, 1), # Treatment group's parameters
       go_cut = 0.2,
       stop_cut = 0.05
-    )
+    ),
+    label = "Assertion on parX failed Element 1 is not >= 2.22507e-308"
```

Comment:
```suggestion
    label = "Assertion on parX failed Element 1 is not >="
```
better omit this specific number because that might be platform specific, so can lead to false errors later on CRAN. Partial matches are usually sufficient for the error messages